tests/test_self.py

```diff
@@ -212,6 +212,11 @@ def test_nonexistent_config_file(self) -> None:
     def test_error_missing_arguments(self) -> None:
         self._runtest([], code=32)
 
+    def test_disable_all(self) -> None:
+        out = StringIO()
+        self._runtest([UNNECESSARY_LAMBDA, "--disable=all"], out=out, code=0)
+        assert "show this help message and exit" in out.getvalue().strip()
```

Comment:
I'm torn on whether to print help in this case.  A user might have unintentionally disabled everything due to the order of command-line arguments.

Reply:
Yeah, what about a simple ``No files to lint, exiting`` ? The help message a is a wall of text and it's not intuitive that there wasn't any files to lint when getting a wall of text..